src/org/etools/j1939_84/controllers/part06/Part06Step03Controller.java

```diff
@@ -86,6 +88,6 @@ protected void run() throws Throwable {
         }
 
         // 6.6.3.2.c Fail if NACK not received from OBD ECUs that did not provide a DM12 message.
-        checkForNACKs(dsPackets, filterAcks(dsResults), "6.6.3.2.b");
+        checkForNACKsFromObdModules(dsPackets, filterAcks(dsResults), "6.6.3.2.b");
```

Comment:
This should be using the other method.  The error with this will be mis-leading as you have it.